### PR TITLE
Update onramp-kit with React example

### PR DESCRIPTION
**Pull request by ademidun**
Show how to use the Onramp Kit in a React App since the onramp kit will primarily be used with a UI context.

Include videos and screenshots to give users a clearer visualization of the Onramp Kit UX.

Related: https://github.com/5afe/safe-space/pull/8

To see how the docs look within the docs.safe.global UI, [View Documentation draft in Gitbook](https://app.gitbook.com/o/-MhyDtGxUyODu7d4sgda/s/-MhyEZtd5TVytPJtyS7v-2910905616/~/revisions/gcQy2vSvqY4UP67DtqrD/learn/safe-core/safe-core-account-abstraction-sdk/onramp-kit)

https://user-images.githubusercontent.com/9806858/228414080-59d2bc99-2323-4143-8736-24dbb175289f.mp4

### CODE REVIEW

**Comment by yagopv** [comment]
The readme video should allow to see the complete purchase in my opinion

File: learn/safe-core/safe-core-account-abstraction-sdk/onramp-kit.md [hunk 1]
```diff
@@ -1,115 +1,209 @@
...
+Once the hackathon and Stripe pilot are over, the server will be removed and you should use your own keys and server if you plan on using the [StripeAdapter](https://github.com/safe-global/account-abstraction-sdk/blob/838d89e98aa9f9e32a6cd499a898fa7f6e69e7c6/packages/onramp-kit/src/adapters/stripe/StripeAdapter.ts).
+
+## Quickstart
 
-Currently this package is only prepared to work with Stripe. See [considerations and limitations](#considerations-and-limitations) for more details.
+In this tutorial, we'll create a React component that allows you to enter an Ethereum address and fund it using a credit card. The full code example can be found in the [Safe Space repo](https://github.com/5afe/safe-space) in the [WalletFund.tsx file](https://github.com/5afe/safe-space/blob/onramp-kit-integration/src/scenes/Wallet/WalletFund.tsx) in [PR #8](https://github.com/5afe/safe-space/pull/8). We will also be using a [pre-deployed server](https://github.com/safe-global/account-abstraction-sdk/tree/main/packages/onramp-kit/example/server).
```
Comment (yagopv) [inline learn/safe-core/safe-core-account-abstraction-sdk/onramp-kit.md:13]: The deployed server is [this one](https://github.com/5afe/aa-stripe-service)

**Comment by yagopv** [comment]
@ademidun after the hackaton we are going to [merge some docs](https://github.com/safe-global/safe-docs/pull/190) referencing an API change so the way these examples work should be updated

File: learn/safe-core/safe-core-account-abstraction-sdk/onramp-kit.md (hunk 1 above)
Comment (ademidun) [reply learn/safe-core/safe-core-account-abstraction-sdk/onramp-kit.md:13]:
```suggestion
In this tutorial, we'll create a React component that allows you to enter an Ethereum address and fund it using a credit card. The full code example can be found in the [Safe Space repo](https://github.com/5afe/safe-space) in the [WalletFund.tsx file](https://github.com/5afe/safe-space/blob/onramp-kit-integration/src/scenes/Wallet/WalletFund.tsx) in [PR #8](https://github.com/5afe/safe-space/pull/8). We will also be using a [pre-deployed server](https://github.com/5afe/aa-stripe-service).
```

File: learn/safe-core/safe-core-account-abstraction-sdk/onramp-kit.md [hunk 2]
```diff
@@ -1,115 +1,209 @@
...
-    onRampBackendUrl: <Your backend url> // You should deploy your own server
+  const fundWallet = async function() {
+        
+    
+    const safeOnRamp = await SafeOnRampKit.init(SafeOnRampProviderType.Stripe, {
+      onRampProviderConfig: {
+        // Get public key from Stripe: https://dashboard.stripe.com/register
+        stripePublicKey:
+        'pk_test_51MZbmZKSn9ArdBimSyl5i8DqfcnlhyhJHD8bF2wKrGkpvNWyPvBAYtE211oHda0X3Ea1n4e9J9nh2JkpC7Sxm5a200Ug9ijfoO',
+        // Deploy your own server: https://github.com/safe-global/account-abstraction-sdk/tree/main/packages/onramp-kit/example/server
```
Comment (ademidun) [inline learn/safe-core/safe-core-account-abstraction-sdk/onramp-kit.md:74]:
```suggestion
        // Deploy your own server: https://github.com/5afe/aa-stripe-service
```

File: learn/safe-core/safe-core-account-abstraction-sdk/onramp-kit.md [hunk 3]
```diff
@@ -1,115 +1,209 @@
...
 ### Prerequisites
 
-- [Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
-- [Stripe account to get your own public and private keys](https://dashboard.stripe.com/register)
-- A deployed server ([example](https://github.com/safe-global/account-abstraction-sdk/tree/main/packages/onramp-kit/example/server)) for communicating with Stripe APIs
+Note: For the duration of the Safe hackathon and Stripe pilot, you can use this tutorial without Steps 2 and 3 for testing purposes.
+
+1. [Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+1. [Stripe account to get your own public and private keys](https://dashboard.stripe.com/register)
+1. A deployed server ([example](https://github.com/safe-global/account-abstraction-sdk/tree/main/packages/onramp-kit/example/server)) for communicating with Stripe APIs
```
Comment (ademidun) [inline learn/safe-core/safe-core-account-abstraction-sdk/onramp-kit.md:31]:
```suggestion
1. A deployed server ([example](https://github.com/5afe/aa-stripe-service)) for communicating with Stripe APIs
```

**Comment by ademidun** [comment]
> ⚠️ @ademidun after the hackaton we are going to [merge some docs](https://github.com/safe-global/safe-docs/pull/190) regarding some API change so the way these examples work should be updated

Sounds good. In the interim, can we merge this so that developers can use it for the hackathon?

File: learn/safe-core/safe-core-account-abstraction-sdk/onramp-kit.md [hunk 4]
```diff
@@ -1,115 +1,209 @@
...
-# OnRamp Kit
+### Demo
+https://user-images.githubusercontent.com/9806858/228415245-7b18b226-5ef4-4b7d-a4b1-5c9c53f625f5.mp4
 
-The [Onramp kit](https://github.com/safe-global/account-abstraction-sdk/tree/main/packages/onramp-kit) allows users to buy cryptocurrencies using a credit card and other payment options.
+<img width="394" alt="Screen Shot 2023-03-29 at 4 34 44 AM" src="https://user-images.githubusercontent.com/9806858/228415363-024fbfcd-443e-4024-8a2f-56390a09e9d4.png">
+
+<img width="388" alt="Screen Shot 2023-03-29 at 4 37 04 AM" src="https://user-images.githubusercontent.com/9806858/228415381-cf838ccc-4002-4327-a654-7c9dc2ba68f7.png">
+
+<img width="748" alt="Screen Shot 2023-03-29 at 4 36 52 AM" src="https://user-images.githubusercontent.com/9806858/228415378-901931d8-7095-4257-a7ae-5ff67246c297.png">
```
Comment (dasanra) [inline learn/safe-core/safe-core-account-abstraction-sdk/onramp-kit.md:22]: Alt should be relevant information describing the image to help screen readers

File: learn/safe-core/safe-core-account-abstraction-sdk/onramp-kit.md [hunk 5]
```diff
@@ -1,115 +1,209 @@
...
+| **Zip Code**                | 12345                       | Use 12345 for zip code                                      |
+| **Test Credit Card Number** | 4242424242424242            | Use test credit card 4242424242424242                       |
+| **Expiration Date**         | 12/24                       | Use future expiration date 12/24                            |
+| **CVC**                     | 123                         | Use any CVC 123                                             |
+| **Billing Zip Code**        | 12345                       | Use any zip code 12345 for billing                          |
+
+### Onramp Kit KYC Test Data - Examples
+<img width="403" alt="Screen Shot 2023-03-29 at 4 32 26 AM" src="https://user-images.githubusercontent.com/9806858/228418052-30b2239a-ca19-4639-9858-4344d3ba7d45.png">
+<img width="392" alt="Screen Shot 2023-03-29 at 4 33 03 AM" src="https://user-images.githubusercontent.com/9806858/228418056-48cfa6a6-fde9-4504-a8be-ce91b03c960f.png">
+<img width="386" alt="Screen Shot 2023-03-29 at 4 34 25 AM" src="https://user-images.githubusercontent.com/9806858/228418059-b83b6357-a6b0-4f09-a4b2-3b89767cb4f0.png">
```
Comment (dasanra) [inline learn/safe-core/safe-core-account-abstraction-sdk/onramp-kit.md:209]: `alt` should describe the image to help screen readers

**Comment by yagopv** [comment]
> Sounds good. In the interim, can we merge this so that developers can use it for the hackathon?

Sure but it ends tomorrow right?

File: learn/safe-core/safe-core-account-abstraction-sdk/onramp-kit.md (hunk 5 above)
Comment (ademidun) [reply learn/safe-core/safe-core-account-abstraction-sdk/onramp-kit.md]: Thanks for the catch. Fixed: https://github.com/safe-global/safe-docs/pull/192/commits/9f225d29932aa7eb56fd4ed54fa74bf600f19228

File: learn/safe-core/safe-core-account-abstraction-sdk/onramp-kit.md (hunk 4 above)
Comment (ademidun) [reply learn/safe-core/safe-core-account-abstraction-sdk/onramp-kit.md]: Thanks for the catch. Fixed: https://github.com/safe-global/safe-docs/pull/192/commits/9f225d29932aa7eb56fd4ed54fa74bf600f19228

**Pull request merged**